modulefiles/ufs_common

```diff
@@ -13,10 +13,10 @@ module load fms/2020.04.03
 module load bacio/2.4.1
 module load crtm/2.3.0
 module load g2/3.4.1
```

Comment:
According to Kyle, we need g2/3.4.2

Reply:
In my offline and inline post testing, g2/3.4.1 was used. Let me test g2/3.4.2 in UPP first.